README.md

```diff
@@ -57,6 +57,7 @@ Registered entry points for aiida.calculations:
 The example folder provides scripts to submit calculations in the calculations folder, and tutorials in jupyter notebook format in the tutorials folder.
 
 A quick demo of how to submit a calculation using the provided example files:
+You might need create a [code](https://aiida.readthedocs.io/projects/aiida-core/en/stable/howto/run_codes.html#how-to-create-a-code) for Janus to be recognised as a code by aiida
 ```shell
 verdi daemon start     # make sure the daemon is running
```

Comment:
This doesn't work with the after just running `verdi presto`, so either need to get it configured before this point, or remove this.